html/getstarted.html

```diff
@@ -0,0 +1,485 @@
+
+	<div></div>
+</div>
+<!-- LANGUAGE-GRID END -->
+
+<!-- COMMAND-GRID START -->
+<div class="grid-command">
+	<div id="f4"></div>
+
+	<div style="width:200px;color:black">Run this command</div>
+	<div id="cmd-wrapper" style="padding:0;margin:0">
+		<textarea class="editor-output" rows="6" cols="20" spellcheck="false" id="cmd-run"
+			style="background-color:rgba(31,54,77,0.3);color:black">
+		</textarea>
+	</div>
+
+	<div></div>
+</div>
+<!-- COMMAND-GRID END -->
+
+<!-- ASSETS START -->
+<center>
+	<div style="max-width:800px">
+		<h3 id="page-assets" style="text-align:left">Get your assets</h3>
+
+		<p align="justify">Here is a summary of the currently available assets you can use depending on your needs:</p>
+
+		<br>
+		<div class="asset-wrapper">
+			<div>Asset</div>
+			<div>OS</div>
+			<div>Arch</div>
+			<div></div>
+		</div>
+
+		<hr style="height:0.03em;border:none;color:#333;background-color:#333;" />
+
+		<div class="asset-wrapper">
+			<div><p>mlpack-3.4.1.msi</p></div>
+			<div><p>Windows</p></div>
+			<div><p>x86|x64</p></div>
+			<div><a href="files/mlpack-3.4.1.msi"><button class="asset-button">Download</button></a></div>
+		</div>
+
+		<hr style="height:0.03em;border:none;color:#333;background-color:#333;" />
+
+		<div class="asset-wrapper">
+			<div><p>mlpack-3.4.1.tar.gz</p></div>
+			<div><p>All</p></div>
+			<div><p>x86|x64</p></div>
+			<div><a href="files/mlpack-3.4.1.tar.gz"><button class="asset-button">Download</button></a></div>
+		</div>
+
+		<hr style="height:0.03em;border:none;color:#333;background-color:#333;" />
+
+		<div class="asset-wrapper">
+			<div><p>mlpack-3.4.1.zip</p></div>
+			<div><p>All</p></div>
+			<div><p>x86|x64</p></div>
+			<div><a href="files/mlpack-3.4.1.zip"><button class="asset-button">Download</button></a></div>
+		</div>
+
+		<hr style="height:0.03em;border:none;color:#333;background-color:#333;" />
+
+		<div class="asset-wrapper">
+			<div><p>mlpack-3.4.1-windows-no-libs.zip</p></div>
+			<div><p>Windows</p></div>
+			<div><p>x86|x64</p></div>
+			<div><a href="files/mlpack-3.4.1-windows-no-libs.zip"><button class="asset-button">Download</button></a></div>
+		</div>
+	</div>
+</center>
+<!-- ASSETS END -->
+
+<!-- COMPILING-MANUALLY START -->
+<center>
+	<div style="max-width:800px;text-align:left;">
+		<h3 id="page-compiling">Compiling manually</h3>
+		<p align="justify">
+			In case none of the binary packages listed on our website work for your system, or you want to
+			modify mlpack, you will need to build it from source.
+		</p>
+
+		<h3 id="page-compiling-linux">Ubuntu, Debian, Fedora, Red Hat, macOS</h3>
+		<a id="built-linux" href="https://www.mlpack.org/doc/mlpack-3.4.1/doxygen/build.html">Linux build tutorial &rarr;</a>
+
+		<h3 id="page-compiling-windows">Windows</h3>
+		<a id="built-windows" href="https://www.mlpack.org/doc/mlpack-3.4.1/doxygen/build_windows.html">Windows build tutorial &rarr;</a>
+		<br>
```

Comment:
Up to you, but this might be more effective as an `ul` instead of `h3`/`a`.

Reply:
Good idea, fixed in https://github.com/mlpack/mlpack.org/pull/26/commits/febbdadc1c87db8c06b290040b7240c9d4394f10.